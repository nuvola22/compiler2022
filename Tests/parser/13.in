function (): Integer;
begin end;

begin end.