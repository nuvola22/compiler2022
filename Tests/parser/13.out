(1, 10), Id expected
